### PR TITLE
feat(Search): add "placeholder" & documentation

### LINKED ISSUE #3806: search placeholder is implemented but not documented
hi.
in "react semantic ui" document
search placeholder API is not documented while is implemented for component
please add
tnx

### PR DESCRIPTION
**Pull request by rohanrav**
Fixes #3806.

### CODE REVIEW

**Comment by rohanrav** [comment]
Here is the issue this PR addresses: #3806

**Review by gbhasha — APPROVED** [review]

File: docs/src/examples/modules/Search/Types/SearchExampleStandard.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ function SearchExampleStandard() {
       <Grid.Column width={6}>
         <Search
           loading={loading}
+          placeholder='Search'
```
Comment (layershifter) [inline docs/src/examples/modules/Search/Types/SearchExampleStandard.js]:
```suggestion
          placeholder='Search...'
```

_nit_ 🐱

**Comment by layershifter** [comment]
@rohanrav great job, I also added a small unit test 🐱

**Comment by noinkling** [comment]
Hmm, this change causes the search's `<input>` element to have a superfluous (no value) `placeholder` attribute when it isn't provided. I would rather avoid that.

I think it might be because of assigning it an empty string in `defaultProps`? That doesn't seem necessary to me anyway.

**Comment by layershifter** [comment]
> Hmm, this change causes the search's `<input>` element to have a superfluous (no value) `placeholder` attribute when it isn't provided. I would rather avoid that.
> 
> I think it might be because of assigning it an empty string in `defaultProps`? That doesn't seem necessary to me anyway.

@noinkling good call, yeah it should be removed from `defaultProps`. I will make a PR.

**Comment by layershifter** [comment]
@noinkling #4327

**Pull request merged**